pom.xml

```diff
@@ -734,6 +735,9 @@
               -Djava.net.preferIPv4Stack=true
               -Djava.awt.headless=true
               -XX:+CMSClassUnloadingEnabled -ea
+              -Djdk.attach.allowAttachSelf=true
+              -Duser.language=en
```

Comment:
This fixes unit tests but opens Drill for bugs on system where language or country are different from en/US.

Reply:
This tests fails on non-english locale. Main reason is a decimal and thousand separator char.
https://drill.apache.org/docs/compiling-drill-from-source/ - I don't see any requirements about locale in documentaion. Maybe it must be an another Bug+PR ?

TestDurationFormat.testCompactSecMillis:58->validateDurationFormat:43 expected:<4[,]545s> but was:<4[.]545s>
TestDurationFormat.testCompactTwoDigitMilliSec:48->validateDurationFormat:43 expected:<0[,]045s> but was:<0[.]045s>
TestFunctionsQuery.testToCharFunction:534 »  After matching 0 records, did not...
TestSelectivity.testFilterSelectivityOptions » UserRemote PARSE ERROR: Encount...
TestVarDecimalFunctions.testCastDecimalDouble:680->BaseTestQuery.testRunAndReturn:341 » Rpc
TestVarDecimalFunctions.testCastDoubleDecimal:650->BaseTestQuery.testRunAndReturn:341 » Rpc
TestVarDecimalFunctions.testDecimalToChar:731 »  at position 0 column '`s1`' m...
TestTopNSchemaChanges.testMissingColumn:192 »  org.apache.drill.common.excepti...
TestTopNSchemaChanges.testNumericTypes:82 »  org.apache.drill.common.exception...
TestTopNSchemaChanges.testUnionTypes:162 »  org.apache.drill.common.exceptions...
TestMergeJoinWithSchemaChanges.testNumericStringTypes:192->BaseTestQuery.testRunAndReturn:341 » Rpc
TestMergeJoinWithSchemaChanges.testNumericTypes:114->BaseTestQuery.testRunAndReturn:341 » Rpc
TestMergeJoinWithSchemaChanges.testOneSideSchemaChanges:348->BaseTestQuery.testRunAndReturn:341 » Rpc
TestExternalSort.testNumericTypesLegacy:49->testNumericTypes:113 »  org.apache...
TestExternalSort.testNumericTypesManaged:44->testNumericTypes:113 »  org.apach...
TestImageRecordReader.testAviImage:101->createAndQuery:50 »  at position 0 col...
TestImageRecordReader.testBmpImage:56->createAndQuery:50 »  at position 0 colu...
TestImageRecordReader.testEpsImage:121->createAndQuery:50 »  at position 0 col...
TestImageRecordReader.testJpegImage:71->createAndQuery:50 »  at position 0 col...
TestImageRecordReader.testMovImage:111->createAndQuery:50 »  at position 0 col...
TestImageRecordReader.testPngImage:81->createAndQuery:50 »  at position 0 colu...
TestImageRecordReader.testPsdImage:86->createAndQuery:50 »  at position 0 colu...